test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -56,7 +56,7 @@ public static void main(String[] argv) throws Exception {
             robot.delay(1000);
 
             SwingUtilities.invokeAndWait(() -> sbar.pressMouse());
-            latch.await();
+            mousePressLatch.await(2, TimeUnit.SECONDS);
```

Comment:
```suggestion
            if (!mousePressLatch.await(2, TimeUnit.SECONDS)) {
                throw new RuntimeException("Timed out waiting for mouse press");
            }
```

If you don't throw the exception, the test will continue to run… Yet we're sure it is likely to fail.

Reply:
ok..modified..